tests/ui_tests/test_header_and_footer.py

```diff
@@ -31,80 +32,121 @@
-    with wait_for_new_window(app_fs.driver):
-        footer.click_version_link_in_footer()
-    with allure.step(f"Check new opened page: {params['docs']}"):
-        BasePageObject(app_fs.driver, app_fs.adcm.url).wait_url_contains_path(params["docs"])
+class TestHeader:
+    def test_check_header_tabs_for_authorised_user(self, app_fs):
+        header = PageHeader(app_fs.driver, app_fs.adcm.url)
+
+        header.click_arenadata_logo_in_header()
+        intro_page = AdminIntroPage(app_fs.driver, app_fs.adcm.url)
+        header.wait_url_contains_path(intro_page.path)
+
+        header.click_cluster_tab_in_header()
+        cluster_page = ClusterListPage(app_fs.driver, app_fs.adcm.url)
+        header.wait_url_contains_path(cluster_page.path)
+
+        header.click_hostproviders_tab_in_header()
+        provider_page = ProviderListPage(app_fs.driver, app_fs.adcm.url)
+        header.wait_url_contains_path(provider_page.path)
+
+        header.click_hosts_tab_in_header()
+        host_page = HostListPage(app_fs.driver, app_fs.adcm.url)
+        header.wait_url_contains_path(host_page.path)
+
+        header.click_jobs_tab_in_header()
+        job_page = JobListPage(app_fs.driver, app_fs.adcm.url)
+        header.wait_url_contains_path(job_page.path)
+
+        header.click_bundles_tab_in_header()
+        bundle_page = BundleListPage(app_fs.driver, app_fs.adcm.url)
+        header.wait_url_contains_path(bundle_page.path)
+
+        header.click_job_block_in_header()
+        job_page = JobListPage(app_fs.driver, app_fs.adcm.url)
+        header.wait_url_contains_path(job_page.path)
+        header.check_job_popup()
+
+    def test_check_header_help_links_for_authorised_user(self, app_fs):
+        params = {"help_link": "t.me/joinchat/", "docs_link": "docs.arenadata.io/adcm/"}
+        header = PageHeader(app_fs.driver, app_fs.adcm.url)
+        header.click_help_button_in_header()
+        header.check_help_popup()
+        with wait_for_new_window(app_fs.driver):
+            header.click_ask_link_in_help_popup()
+        with allure.step(f"Check new opened page: {params['help_link']}"):
+            BasePageObject(app_fs.driver, app_fs.adcm.url).wait_url_contains_path(params["help_link"])
+            close_current_tab(app_fs.driver)
+        header.click_help_button_in_header()
+        with wait_for_new_window(app_fs.driver):
+            header.click_doc_link_in_help_popup()
+        with allure.step(f"Check new opened page: {params['docs_link']}"):
+            BasePageObject(app_fs.driver, app_fs.adcm.url).wait_url_contains_path(params["docs_link"])
+
+    def test_check_header_user_settings_for_authorised_user(self, app_fs):
+        header = PageHeader(app_fs.driver, app_fs.adcm.url)
+        header.click_account_button_in_header()
+        header.check_account_popup()
+        header.click_settings_link_in_acc_popup()
+        header.wait_url_contains_path(AdminSettingsPage(app_fs.driver, app_fs.adcm.url).path)
```

Comment:
Why like that and not wait until the page is opened?

Reply:
it was the first implementation and now I just moved these tests into class and didn't change inside

Reply:
Got it